app/src/main/java/chat/rocket/android/fragment/sidebar/SidebarMainFragment.java

```diff
@@ -10,19 +10,7 @@
 import android.support.v7.widget.SearchView;
 import android.view.View;
 import android.widget.CompoundButton;
-import android.widget.ImageView;
```

Comment:
Just for future reference, don't need to change now, try to avoid reordering imports alongside other changes in the file., it is harder for reviewers to see what imports where added or removed, if any.

I'll be creating a default config and coding style, including imports ordering, and will do the reordering in a single PR just with that.